lib/syntax.ml

```diff
@@ -11,6 +11,7 @@ type env = {
   tables : Tables.table list;
   schema : table_name Schema.Source.t;
   insert_schema : Schema.t;
+  param_strict: bool;
```

Comment:
need comment what is the meaning of this flag
afaics it is used apply non-null comparison semantics in SELECT/UPDATE/DELETE only

Reply:
added comment